### ISSUE TITLE
No title bar

### ISSUE THREAD

**Issue by nemith** [marten-de-vries/mozapp#1]
Because the tab bar is hidden there seems to be no title bar either which means there is no way to open/close, move the resulting app.  

Firefox 96.0.3 on Fedora X11

**Comment by marten-de-vries** [comment]
Yes, this is an issue. In Gnome you can drag windows around using the mouse when pressing the Super (i.e. Windows) key (although it's configurable). Alt+F4, Ctrl+Q or Ctrl+W will close the app. Super+H will minimize a window. Other desktop environments probably have similar functions.

Those workarounds are good enough for my usage, especially as I use the 'Unite' gnome extension to hide the menu bar a lot of the time anyway. That said, if you find [the right user chrome css](https://github.com/marten-de-vries/mozapp/blob/main/mozapp.py#L18) to show a menu bar but nothing else, I'd be happy to merge it.
